app/views/providers/legal_aid_applications/_legal_aid_applications.html.erb

```diff
@@ -26,5 +26,11 @@
         </tbody>
       <% end %>
     </table>
+    <% if local_assigns[:pagy] %>
```

Comment:
will this condition ever be false?

Reply:
I'm conscious that this is partial that could be used elsewhere. I thought it safer to put this in place.